src/Sylius/Component/Core/Model/Order.php

```diff
@@ -472,11 +478,18 @@ public function getNonDiscountedItemsTotal(): int
         return $total;
     }
 
-    public function getCreatedByGuest(): bool
+    public function isCreatedByGuest(): bool
     {
         return $this->createdByGuest;
     }
 
+    public function getCreatedByGuest(): bool
+    {
+        @trigger_error('This method is deprecated since Sylius 1.11 and it will be removed in Sylius 2.0. Please use `isCreatedByGuest` instead.');
```

Comment:
```suggestion
        @trigger_error('This method is deprecated since Sylius 1.12 and it will be removed in Sylius 2.0. Please use `isCreatedByGuest` instead.');
```

Reply:
And add this note also to an UPGRADE file